connman/client/commands.c

```diff
@@ -639,6 +652,17 @@ static int cmd_tether(char *args[], int num, struct connman_option *options)
 	return tether_set(args[1], set_tethering);
 }
 
+static int cmd_tethering_clients(char *args[], int num, struct connman_option *options)
```

Comment:
args & options are not used. Maybe touching those variables.

Reply:
Same here, function prototype requires. Should not mess with it.